sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1978,36 +1978,26 @@ public enum SearchMode
     public static partial class SearchModelFactory
     {
         public static Azure.Search.Documents.Indexes.Models.AnalyzedTokenInfo AnalyzedTokenInfo(string token, int startOffset, int endOffset, int position) { throw null; }
-        public static Azure.Search.Documents.Models.AutocompleteResults AutocompleteResults(double? coverage, System.Collections.Generic.IReadOnlyList<Azure.Search.Documents.Models.Autocompletion> results) { throw null; }
-        public static Azure.Search.Documents.Models.Autocompletion Autocompletion(string text, string queryPlusText) { throw null; }
-        public static Azure.Search.Documents.Indexes.Models.CharFilter CharFilter(string oDataType, string name) { throw null; }
-        public static Azure.Search.Documents.Indexes.Models.CognitiveServicesAccount CognitiveServicesAccount(string oDataType, string description) { throw null; }
-        public static Azure.Search.Documents.Indexes.Models.DataChangeDetectionPolicy DataChangeDetectionPolicy(string oDataType) { throw null; }
-        public static Azure.Search.Documents.Indexes.Models.DataDeletionDetectionPolicy DataDeletionDetectionPolicy(string oDataType) { throw null; }
+        public static Azure.Search.Documents.Models.AutocompleteResults AutocompleteResults(double? coverage, System.Collections.Generic.IReadOnlyList<Azure.Search.Documents.Models.AutocompleteItem> results) { throw null; }
+        public static Azure.Search.Documents.Models.AutocompleteItem Autocompletion(string text, string queryPlusText) { throw null; }
```

Comment:
I missed this before (or maybe it was post-merge with your rename), but the method name should be `AutocompleteItem` now.